src/Build/Definition/ProjectCollection.cs

```diff
@@ -26,6 +26,7 @@
 using InvalidProjectFileException = Microsoft.Build.Exceptions.InvalidProjectFileException;
 using LoggerMode = Microsoft.Build.BackEnd.Logging.LoggerMode;
 using ObjectModel = System.Collections.ObjectModel;
+using Microsoft.Build.BuildEngine;
```

Comment:
This change broke the build and added:
`error NU1702: (NETCORE_ENGINEERING_TELEMETRY=Build) ProjectReference '/home/vsts/work/1/s/src/Deprecated/Engine/Microsoft.Build.Engine.csproj' was resolved using '.NETFramework,Version=v4.7.2' instead of the project target framework '.NETCoreApp,Version=v6.0'. This project may not be fully compatible with your project.`